app/src/androidTest/java/fiofoundation/io/fiokotlinsdktestapp/TestNetSdkTests.kt

```diff
@@ -29,17 +30,17 @@ class TestNetSdkTests {
 
     private val baseUrl = "https://testnet.fioprotocol.io:443/v1/"
 
-    private var alicePrivateKey = "your private key"
```

Comment:
Setting the key variables to "your ... key" was put there to remind the developer what needs to go there to run the tests. Why remove it?